src/main/java/com/algorand/algosdk/v2/client/model/TransactionParametersResponse.java

```diff
@@ -0,0 +1,69 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.Objects;
+
+import com.algorand.algosdk.util.Encoder;
+import com.algorand.algosdk.v2.client.common.PathResponse;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TransactionParametersResponse extends PathResponse {
+
+	/**
+	 * ConsensusVersion indicates the consensus protocol version 
+	 * as of LastRound. 
+	 */	@JsonProperty("consensus-version")
+	public String consensusVersion;
+
+	/**
+	 * Fee is the suggested transaction fee 
+	 * Fee is in units of micro-Algos per byte. 
+	 * Fee may fall to zero but transactions must still have a fee of 
+	 * at least MinTxnFee for the current network protocol. 
+	 */	@JsonProperty("fee")
+	public Long fee;
+
+	/**
+	 * GenesisHash is the hash of the genesis block. 
+	 */ @JsonProperty("genesis-hash")
+	public void genesisHash(String base64Encoded) {
+		 this.genesisHash = Encoder.decodeFromBase64(base64Encoded);
+	 }
+	 @JsonProperty("genesis-hash")
+	 public String genesisHash() {
+		 return Encoder.encodeToBase64(this.genesisHash);
```

Comment:
How does this decide the type is a byte array, and the return type is a string? I think this is another case where `Digest` is the right type.

Reply:
Digest will be resolved later.